htdocs/themes/default/modules/system/system_siteclosed.tpl

```diff
@@ -41,12 +41,12 @@
             <!-- Start Header -->
             <table cellspacing="0">
                 <tr id="header">
-                    <td id="headerlogo"><a href="<{xoAppUrl /}>" title="<{$xoops_sitename}>"><img src="<{xoImgUrl 'xoops-logo.png'}>"
+                    <td id="headerlogo"><a href="<{xoAppUrl '/'}>" title="<{$xoops_sitename}>"><img src="<{xoImgUrl 'xoops-logo.png'}>"
                                                                                                   alt="<{$xoops_sitename}>"/></a></td>
                     <td id="headerbanner"><{$xoops_banner}></td>
                     <td id="xo-userbar_siteclosed">
                         <!-- menu in anonymous mode  -->
-                        <form method="post" action="<{xoAppUrl /user.php op=login}>">
+                        <form method="post" action="<{xoAppUrl '/user.php op=login'}>">
```

Comment:
I think the correct syntax would be ```"<{xoAppUrl '/user.php?op=login'}>``` with a question mark introducing the query string.